content/pages/tools/jupyter-book.html

```diff
@@ -0,0 +1,137 @@
+{#
+  View this page at http://localhost:8080/tools/jupyter-book/
+#}
+<html>
+  <head>
+    <title>Publish your book on the internet with Read the Docs</title>
+    <meta name="template" content="page" />
+    <meta name="slug" content="tools/jupyter-book" />
+    <meta name="status" content="hidden" />
+    <meta name="description" content="Create an HTML version of your book and publish it online." />
+  </head>
+  <body>
+
+{% import "partials/basic.html" as basic %}
+{% import "partials/marketing.html" as marketing %}
+{% import "partials/technical.html" as technical %}
+{% import "partials/homepage.html" as homepage %}
+
+<section>
+
+  {% call basic.header_image(
+    header="Publish your book on the internet",
+    image="/images/jupyter-book-logo.png",
+    image_alt="Jupyter Book logo",
+    vertical_align="middle")
+  %}
+    <p>
+        Jupyter Book works hand by hand with Read the Docs to allow your book to be published online
+        and always available to your readers anytime they need it.
+    </p>
+
+    <p class="ui basic padded vertical segment">
+      <a class="ui large teal stackable button"
+      data-analytics="signup-modal"
+       onclick="jQuery('#signup-modal').modal('show');">
+        <i class="fad fa-play icon"></i>
+        Create an account
+      </a>
+      <a href="https://example-jupyter-book.readthedocs.io/" class="ui large stackable button">
+        See a demo
+      </a>
+    </p>
+  {% endcall %}
+</section>
+
+<section id="features">
+  {% call marketing.highlight_list() %}
+    {% call marketing.highlight_item(
+        icon="olive fa-people-arrows",
+        icon_style="--fa-secondary-color: violet; --fa-secondary-opacity: 1;",
+        header="Pair-review the changes with deploy previews",
```

Comment:
I think we've been using `build preview` or `pull request preview` for this.

```suggestion
        header="Pair-review the changes with build previews",
```